content/docs/replication/deployment/powerflex.md

```diff
@@ -257,6 +259,18 @@ replication enabled Storage Classes. The CSI PowerFlex driver will create a
 volume on the array, add it to a VolumeGroup and configure replication using the
 parameters provided in the replication enabled Storage Class.
 
+### Deleting Replication Groups
+
+To delete a replication group from a cluster, the group must be empty. This
+means that all associated replication pairs must have been deleted/removed.
+
+This command will delete the empty replication group from the cluster and the
+associated replication consistency group from the PowerFlex array.
+
+```shell
+  kubectl delete rg <rg-id>
+```
+
```

Comment:
Can this added to tools.md, Its not specific to powerfkex
@santhoshatdell - Let me know your comments ?

Reply:
I agree. Delete RG is not specific to PowerFlex. It can be moved to a common place.
Tools.md seems to be repctl specific. How about replication-actions page or some other platform generic page ?

Reply:
Moved to `replication-actions` since `tools` is more focused on `repctl` actions.